src/main/java/org/broadinstitute/hellbender/engine/samples/Affection.java

```diff
@@ -0,0 +1,22 @@
+package org.broadinstitute.hellbender.engine.samples;
```

Comment:
I think this `samples` package would be better under `hellbender.utils` rather than `hellbender.engine`